tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -287,6 +293,150 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 		oa.StopInsertDataTo(&clusterA)
 	})
 
+	ginkgo.It("Adhoc backup and restore with BR CRD", func() {
+		if framework.TestContext.Provider != "aws" {
+			framework.Skipf("provider is not aws, skipping")
+		}
+		tcNameFrom := "backup"
+		tcNameTo := "restore"
+		serviceAccountName := "tidb-backup-manager"
+		backupFolder := time.Now().Format(time.RFC3339)
+
+		// create backup cluster
+		tcFrom := fixture.GetTidbCluster(ns, tcNameFrom, "v4.0.0-beta.1")
```

Comment:
Add a version into this package: https://github.com/pingcap/tidb-operator/blob/38d2a865273e1af8d4e6d38d3162aea4d28305cf/tests/e2e/util/image/image.go#L30-L42 (note that TiDB V4 version must be appended into versions in `ListImages`)

We manage all tidb versions in this pkg to do some optimizations in e2e (e.g. use the same version as possible as we can to avoid pulling similar tidb versions).

Reply:
done